examples/cluster-outage-simulation/variables.tf

```
@@ -0,0 +1,31 @@
+variable "public_key" {
+  type        = string
+  description = "Public Programmatic API key to authenticate to Atlas"
+}
+variable "private_key" {
+  type        = string
+  description = "Private Programmatic API key to authenticate to Atlas"
+}
+variable "project_id" {
+  type        = string
+  description = "MongoDB Project ID"
+}
+variable "atlas_cluster_name" {
+  type        = string
+  description = "Atlas Cluster Name that will undergo outage simulation"
+  default     = "Cluster0"
+}
+variable "atlas_cluster_type" {
+  type        = string
+  description = "Atlas Cluster Name that will undergo outage simulation"
```

should this be Type?
```suggestion
  description = "Atlas Cluster Type that will undergo outage simulation"
```